base/templates/base/blocks/duo_img.html

```diff
@@ -14,7 +14,7 @@
     </div>
     <div class="col-xs-12 col-sm-6 collex-duo">
         <figure>
-            <a href="{{original_two.url}}" data-toggle="lightbox"><img class="img-responsive" src="{{original_two.url}}"></a>
+            <a href="{{original_two.url}}" data-toggle="lightbox"><img class="img-responsive" src="{{original_two.url}}" alt="{% firstof value.alt_text value.image_two.alt_text %}"></a>
```

Comment:
Same as above.

Reply:
See above: does not work without `firstof` in the current structure.

Reply:
This has been updated to `alt="{{value.image_two.alt_text}}`